src/data/markdown/docs/20 jslib/01 jslib/05 aws.md

```diff
@@ -4,11 +4,12 @@ excerpt: "aws is a library implementing APIs for accessing a selection of AWS se
 description: "aws is a library implementing APIs for accessing a selection of AWS servicese"
 ---
 
-The `aws` module is a JavaScript library that wraps around some Amazon AWS service APIs. 
+The `aws` module is a JavaScript library that wraps around some Amazon AWS services API. 
 
 The library exposes a couple of configuration and client classes allowing to interact with a subset of AWS services in the context of k6 load test scripts:
 - The [S3Client](/javascript-api/jslib/aws/s3client) to list S3 buckets and the objects they contain, as well as uploading, downloading and deleting objects from them.
 - The [SecretsManagerClient](/javascript-api/jslib/aws/secretsmanagerclient) class allows listing, getting, creating, updating, and deleting secrets from the AWS secrets manager service.
+- The [KMSClient](/javascript-api/jslib/aws/kmsclient) class allows listing keys, and generating data keys from the AWS Key Management Service.
```

Comment:
```suggestion
- The [KMSClient](/javascript-api/jslib/aws/kmsclient) class allows listing keys and generating data keys from the AWS Key Management Service.
```

Reply:
Does the class _allow_, or _do_ these actions? If listing and generating keys is functionality, I'd go with:

> The KMSClient class can list keys and generate data keys from the...

And if "keys" and "data" keys are the same, we could go shorter:

> The KMSClient class lists and generates keys from the ...

Reply:
Oh, now I see that "allows *ing" is everywhere; maybe it's not worth changing.

Reply:
That's really good input. Vale always complains that I use "allow" too much, but I never know what to replace it with 👍🏻